livedata/src/main/java/io/getstream/chat/android/livedata/request/AnyChannelPaginationRequest.kt

```diff
@@ -30,11 +30,19 @@ internal class AnyChannelPaginationRequest(var messageLimit: Int = 30) {
         return messageFilterDirection == null
     }
 
+    fun isRequestingMoreThanLastMessage(): Boolean {
+        return (isFirstPage() && messageLimit > 1) || (isNotFirstPage() && messageLimit > 0)
+    }
```

Comment:
The two new functions added to this class are good candidates for extension functions instead of members, since they just give us "views" over the publicly exposed data of the class.